static/src/stylesheets/module/content-garnett/live-blog/_live-blog.head.scss

```diff
@@ -908,3 +913,43 @@ $pillars: (
         padding-left: $gs-gutter / 2;
     }
 }
+
+// Dead blog overrides
+@each $pillar, $palette in $pillars {
+    .content--liveblog.tonal--tone-dead.content--pillar-#{$pillar} {
+        .tonal__main--tone-dead {
+            @include mq(tablet) {
+                padding-top: $gs-gutter;
+            }
+        }
+
+        .content__head,
+        .content__standfirst {
+            color: $garnett-neutral-1;
+        }
+
+        .content__section-label__link {
+            color: map-get($palette, altKickerColor);
+        }
+
+        .content__series-label__link {
+            color: $garnett-neutral-6;
+        }
+
+        .content__header {
+            background: $garnett-neutral-5;
+        }
+
+        .tonal__standfirst {
+            background: $live-garnett-dead-background;
+        }
+
+        .content__standfirst a {
+            color: map-get($palette, altKickerColor);
+        }
+
+        .tonal__standfirst ul > li::before {
+            background: rgba(51, 51, 51, .4);
```

Comment:
Don't know if it really matters, but this can be written `rgba($garnett-neutral-1, .4)`